src/main/java/mate/academy/model/User.java

```diff
@@ -0,0 +1,58 @@
+package mate.academy.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "users")
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String email;
```

Comment:
```suggestion
 @Column(unique = true)
    private String email;
```
email should be unique

Reply:
I check it in register() method and throw an exception. Also, this annotation does not allow to use an injector, I have problems in the stack.